google/cloud/aiplatform/model_evaluation/model_evaluation_job.py

```diff
@@ -0,0 +1,236 @@
+            model=aiplatform.Model(model_name="..."),
+            prediction_type="classification",
+            pipeline_root="gs://my-pipeline-bucket/runpath",
+            gcs_source_uris=["gs://test-prediction-data"],
+            class_names=["cat", "dog"],
+            target_column_name=["animal_type"],
+            instances_format="jsonl",
+        )
+        my_evaluation = ModelEvaluationJob.submit(
+            model=aiplatform.Model(model_name="..."),
+            prediction_type="regression",
+            pipeline_root="gs://my-pipeline-bucket/runpath",
+            gcs_source_uris=["gs://test-prediction-data"],
+            target_column_name=["price"],
+            instances_format="jsonl",
+        )
+        Args:
+            model_name (str):
+                Required. A fully-qualified model resource name or model ID to run the evaluation
+                job on. Example: "projects/123/locations/us-central1/models/456" or
+                "456" when project and location are initialized or passed.
+            prediction_type (str):
+                Required. The type of prediction performed by the Model. One of "classification" or "regression".
+            pipeline_root (str):
+                Required. The GCS directory to store output from the model evaluation PipelineJob.
+            gcs_source_uris (List[str]):
+                Required. A list of GCS URIs containing your input data for batch prediction. This is used to provide
+                ground truth for each prediction instance, and should include a label column with the ground truth value.
+            target_column_name (str):
+                Required. The name of your prediction column.
+            class_names (List[str]):
+                Required when `prediction_type` is "classification". A list of all possible class names
+                for your classification model's output, in the same order as they appear in the batch prediction input file.
+            instances_format (str):
+                The format in which instances are given, must be one of the Model's supportedInputStorageFormats. If not set, defaults to "jsonl".
+            service_account (str):
+                Specifies the service account for workload run-as account.
+                Users submitting jobs must have act-as permission on this run-as account.
+            network (str):
+                The full name of the Compute Engine network to which the job
+                should be peered. For example, projects/12345/global/networks/myVPC.
+                Private services access must already be configured for the network.
+                If left unspecified, the job is not peered with any network.
+        Returns:
+            model: Updated model resource.
+        Raises:
+            ValueError: If `labels` is not the correct format.
+        """
+
+        if not display_name:
+            display_name = cls._generate_display_name()
+
+        template_params = {
```

Comment:
TODO: does this capture all parameters required by the eval template?

Reply:
I believe the option "dataflow_use_public_ips" needs to be opened up as well, as someone using VPC-SC needs this option along with "network".